oidc-api/src/main/java/uk/gov/di/authentication/oidc/lambda/AuthenticationCallbackHandler.java

```diff
@@ -486,11 +484,15 @@ private Map<String, String> buildDimensions(
             LOG.info(
                     "No mfa method to set. User is either authenticated or signing in from a low level service");
         }
-        VectorOfTrust vtr = clientSession.getVtrWithLowestCredentialTrustLevel();
-        var mfaRequired = !vtr.getCredentialTrustLevel().equals(CredentialTrustLevel.LOW_LEVEL);
+        var orderedVtrList = VectorOfTrust.orderVtrList(clientSession.getVtrList());
+        var mfaRequired =
+                !orderedVtrList
+                        .get(0)
+                        .getCredentialTrustLevel()
+                        .equals(CredentialTrustLevel.LOW_LEVEL);
         var levelOfConfidence = LevelOfConfidence.NONE.getValue();
-        if (vtr.containsLevelOfConfidence()) {
-            levelOfConfidence = vtr.getLevelOfConfidence().getValue();
+        if (orderedVtrList.get(0).containsLevelOfConfidence()) {
+            levelOfConfidence = orderedVtrList.get(0).getLevelOfConfidence().getValue();
```

Comment:
This doesn't look right to me. We spoke about a sorted, comma separated list of the requested values

Reply:
Ahh okay, I put that in the method logging but shall put it here too